### PR TITLE
WIP Try to fix crash when switching monitors

### PR DESCRIPTION
**Pull request by m-g-r**
DO NOT MERGE!

This is WIP trying to debug issue #763 see comment https://github.com/stumpwm/stumpwm/issues/763#issuecomment-619021716

(also contains some other things helping me debugging it and that will probably get to be in separate PRs

update: that stuff is now factored out in PRs #770 #771 #772 #773)

### CODE REVIEW

**Comment by m-g-r** [comment]
This seems to be running quite stable I have to say...

**Comment by dmb2** [comment]
When you get this straightened around, please open one PR with the bug fix and nothing else.

**Comment by m-g-r** [comment]
Yes, it was just for having a diff view for issue #763. I am running this code now on my machine and it prevents a crash and stumpwm continues to run fine. still i am closing this now.

File: tile-group.lisp [hunk 1]
```diff
@@ -31,6 +31,12 @@
    (last-frame :initform nil :accessor tile-group-last-frame)
    (current-frame :accessor tile-group-current-frame)))
 
+(defmethod tile-group-current-frame :before ((group tile-group))
+  (declare (optimize (speed 0) (debug 3) (safety 3)))
```
Comment (PuercoPop) [inline tile-group.lisp:35]: Protip: For debugging purposes you can add `(restrict-compiler-policy 'debug 3)` to your `.sbclrc` so you don't have to be specific as to which function to you to ensure to have been compiled with a high debug setting.

Comment (m-g-r) [reply tile-group.lisp:35]: thanks I've missed those two declarations. removed now in https://github.com/m-g-r/stumpwm/pull/1

**Pull request closed without merging**